src/pocketmine/resourcepacks/BehaviorPack.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/*
+ *
+ *  ____            _        _   __  __ _                  __  __ ____
+ * |  _ \ ___   ___| | _____| |_|  \/  (_)_ __   ___      |  \/  |  _ \
+ * | |_) / _ \ / __| |/ / _ \ __| |\/| | | '_ \ / _ \_____| |\/| | |_) |
+ * |  __/ (_) | (__|   <  __/ |_| |  | | | | | |  __/_____| |  | |  __/
+ * |_|   \___/ \___|_|\_\___|\__|_|  |_|_|_| |_|\___|     |_|  |_|_|
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * @author PocketMine Team
+ * @link http://www.pocketmine.net/
+ *
+ *
+*/
+
+declare(strict_types=1);
+
+
+namespace pocketmine\resourcepacks;
+
```

Comment:
Im not sure if its github or not but you seem to have double whitespace here? One is enough.

Reply:
The project seems to be inconsistent with this. 
Some files like [Event.php](https://github.com/pmmp/PocketMine-MP/blob/stable/src/pocketmine/event/Event.php#L22-L31) use single line, while some like [ResourcePack.php](https://github.com/pmmp/PocketMine-MP/blob/stable/src/pocketmine/resourcepacks/ResourcePack.php#L22-L27) use two.